.github/workflows/ci_build_test.yml

```diff
@@ -34,10 +34,7 @@ jobs:
           command: build
           args: --verbose --workspace
       - name: Cargo Test
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --verbose --workspace
+        run: cargo test --verbose --workspace -- -Z unstable-options
```

Comment:
Think the error you're getting in the CI seems similar to https://github.com/partiql/partiql-lang-rust/pull/322/files.

Might need to use the nightly toolchain?

Reply:
+1, submitted a new commit, let's see.

Reply:
Actually, nightly might not be needed for the build/test. Think you'll need it for the coverage step though.

I'd update the rust toolchain to `dtolnay/rust-toolchain` (like in [partiql-lang-rust](https://github.com/partiql/partiql-lang-rust/blob/main/.github/workflows/ci_build_test.yml#L32-L36)) which is still maintained.

Reply:
Looks like with the new changes we're getting the successful builds; but I think it's better to make it consistent w/the other project. Let me do that to see the outcome.